.devcontainer/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:lts-alpine
+
+ENV NODE_ENV production yarn build
```

Comment:
that's not doing what you think it's doing, ENV sets an env var and that's it. In your case `NODE_ENV="production yarn build"`

Reply:
removed: https://github.com/thelounge/thelounge/pull/4514/commits/0bb0e2f10baaee9471f31d73ec159027768a4d5e